### PR TITLE
add merchant_account_id to authorize transaction

### PR DESCRIPTION
**Pull request by ufchrisg**
- Adds `merchant_account_id` to authorize_transaction view
- Updates ruby version to 2.6.5

### CODE REVIEW

File: lib/conduit/braintree/views/authorize_transaction.erb [hunk 1]
```diff
@@ -4,7 +4,8 @@ MultiJson.dump(
     PaymentMethodToken: token,
     Amount: amount,
     TransactionSource: transaction_source,
-    Descriptor: descriptor
+    Descriptor: descriptor,
+    MerchantAccount: merchant_account_id
```
Comment (pedromartinez) [inline lib/conduit/braintree/views/authorize_transaction.erb]: All this does is add logging. I don't think that's your intention.

File: spec/actions/authorize_transaction_spec.rb [hunk 2]
```diff
@@ -20,18 +20,6 @@
   let(:amount) { 10 }
 
   describe "#perform" do
-    context "with a successful authorization" do
-      let(:mock_status) { "success" }
-      its(:response_status)    { should eql mock_status }
-      its(:transaction_status) { should eql "authorized" }
-      its(:transaction_amount) { should eql amount }
-
-      it "request should contain transaction_source" do
```
Comment (pedromartinez) [inline spec/actions/authorize_transaction_spec.rb]: Technically if the "view" layer is broken, we won't have a test to catch it.

File: lib/conduit/braintree/views/authorize_transaction.erb (hunk 1 above)
Comment (dougperez) [reply lib/conduit/braintree/views/authorize_transaction.erb]: Its already added in the action. Is it not being sent?

Comment (pedromartinez) [reply lib/conduit/braintree/views/authorize_transaction.erb]: Okay, so this PR is actually just for logging then. If so, I'm good with it.

**Pull request merged**